src/AzureDeprecation.Notifications/Functions/EventGridNotificationsFunction.cs

```diff
@@ -32,16 +38,23 @@ public async Task Run([ServiceBusTrigger("new-deprecation-notices", "event-grid-
                 .UsingAuthenticationKey(eventGridAuthKey)
                 .Build();
             
-            var @event = new CloudEvent(CloudEventsSpecVersion.V1_0,
+            var @event = new CloudEvent(
+                CloudEventsSpecVersion.V1_0,
                 "NewDeprecationNoticePublishedV1",
-                new Uri("https://github.com/azure-deprecation/dashboard"),
-                subject: $"/{newDeprecationNoticePublishedV1Message.DeprecationInfo.Impact.Services.First()}")
+                new Uri("https://github.com/azure-deprecation/dashboard"), // TODO: dev/staging dashboard
```

Comment:
Meh, I think this is OK :D but we can make it configurable if you want

Reply:
Not immediately maybe, just saw that we'll be linking **the** dashboard from whatever db this ends up being stored in.